tools/github-issues/Azure.Sdk.Tools.GitHubIssues/Email/EmailSender.cs

```diff
@@ -33,13 +33,7 @@ public static void SendEmail(string emailToken, string from, string template, st
             message.SetSubject($"GitHub Report: {title}");
             message.AddContent(MimeType.Html, template);
 
-
-#if !DEBUG
-            // Don't accidentally send email
             var emailResult = client.SendEmailAsync(message).GetAwaiter().GetResult();
-#else
-            System.IO.File.WriteAllText("output.html", template);
-#endif
```

Comment:
Is there a good way now to run this locally and not worry about accidentally sending a ton of email?

Reply:
I can add this back in, although when I'm debugging services what I typically do is run against the staging configuration environment which only has configuration to send e-mail to me.

Reply:
Put this back in.